plugins/source/aws/codegen/recipes/base.go

```diff
@@ -82,6 +82,26 @@ func awsNameTransformer(f reflect.StructField) (string, error) {
 	return c.ToSnake(f.Name), nil
 }
 
+func awsResolverTransformer(f reflect.StructField, path string) (string, error) {
+	if f.Type.String() == "[]types.Tag" {
+		if path == "Tags" {
+			return "client.ResolveTags", nil
+		}
+		return `client.ResolveTagField("` + path + `")`, nil
+	}
+
+	if path == "Tags" || path == "TagSet" {
+		switch f.Type.String() {
+		case "map[string]string", "map[string]*string", "map[string]interface {}", "[]types.TagDescription":
```

Comment:
1. `"map[string]interface {}"` -> `"map[string]interface{}"` ?
2. maybe use reflect data instead?

Reply:
1. This is what gives
2. It would work here as these are std types, and TagDescription is just from AutoScalingGroups... but is it necessary from a readability standpoint?